dashboard/app/models/contact_rollups_raw.rb

```diff
@@ -15,18 +15,20 @@ class ContactRollupsRaw < ApplicationRecord
   self.table_name = 'contact_rollups_raw'
 
   def self.extract_email_preferences
-    query = get_extraction_query('email_preferences', 'email', ['opt_in'])
+    select_query = 'SELECT email, opt_in, updated_at FROM email_preferences'
+    query = get_extraction_query('dashboard.email_preferences', select_query, 'opt_in')
     ContactRollupsV2.execute_query_in_transaction(query)
   end
 
   def self.extract_parent_emails
     source_sql = <<~SQL
-      SELECT parent_email, MAX(updated_at) AS updated_at
+      SELECT parent_email AS email, 1 AS parent_email, MAX(updated_at) AS updated_at
```

Comment:
Is the `1 AS parent_email` just a placeholder? I guess I see what you're doing here (trying to make them method signature describe what it is extracting), but maybe a comment to describe that this is a unique case and a bit of a hack?

Reply:
Will rename the field from `parent_email` to `is_parent`. It's a data point we get for free so I just take it, even though not sure how it will be used yet 😅.